src/OpenTelemetry/Logs/OpenTelemetryLoggerOptions.cs

```diff
@@ -17,14 +17,28 @@
 #if NET461 || NETSTANDARD2_0
 using System;
 using System.Collections.Generic;
-using Microsoft.Extensions.Logging;
 
 namespace OpenTelemetry.Logs
 {
     public class OpenTelemetryLoggerOptions
     {
         internal readonly List<BaseProcessor<LogRecord>> Processors = new List<BaseProcessor<LogRecord>>();
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not log scopes should be included on generated <see cref="LogRecord"/>s. Default value: False.
+        /// </summary>
+        public bool IncludeScopes { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not log message should be included on generated <see cref="LogRecord"/>s. Default value: False.
+        /// </summary>
+        public bool IncludeMessage { get; set; }
```

Comment:
```suggestion
        public bool IncludeFormattedMessage { get; set; }
```

I think IncludeFormattedMessage is a better name.

Reply:
Sounds good. What about `LogRecord.Message` you want that to be `LogRecord.FormattedMessage` or leave as `LogRecord.Message`?